.buildkite/pipeline.build.yml

```diff
@@ -484,6 +484,18 @@
       --test_env=CONDA_DEFAULT_ENV
       python/ray/tests/...
 
+- label: ":python: Ray on Spark Test"
```

Comment:
Do we need to create a separate test suite, or just be part of small/large python tests?

Reply:
I prefer to create a separate label, because this is an extension plugin, and it might run a bit longer than other tests.